packages/database/src/migrations/20210205033535-AddSTRIVEMolecularDataOverlays-modifies-data.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+import { insertObject } from '../utilities';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const BASE_OVERLAY = {
+  userGroup: 'STRIVE User',
+  dataElementCode: 'value',
+  isDataRegional: true,
+  measureBuilder: 'countEventsPerOrgUnit',
+  presentationOptions: {
+    values: [
+      {
+        value: 'other',
+        color: 'blue',
+      },
+      {
+        value: null,
+        color: 'grey',
+      },
+    ],
+    displayType: 'radius',
+    measureLevel: 'Facility',
+    hideFromLegend: true,
+  },
+  countryCodes: '{PG}',
+  projectCodes: '{strive,explore}',
+};
+
+const STR_QMAL05_OVERLAY = {
+  ...BASE_OVERLAY,
+  id: generateId(),
```

Comment:
Generally the map overlays have more expressive ids than this. Your last one was good (`PG_STRIVE_K13_C580Y_Positive`)